test/test_examples_2d.jl

```diff
@@ -224,6 +280,14 @@ end
   @test_nowarn println(callbacks)
 end
 
+# Only run extended tests if environment variable is set
+if haskey(ENV, "TRIXI_TEST_EXTENDED") && lowercase(ENV["TRIXI_TEST_EXTENDED"]) in ("1", "on", "yes")
+  @testset "Examples (long execution time)" begin
+    @test_nowarn test_trixi_include(joinpath(EXAMPLES_DIR, "elixir_euler_blob_mortar_split_shockcapturing.jl"))
+    @test_nowarn test_trixi_include(joinpath(EXAMPLES_DIR, "elixir_euler_blob_split_shockcapturing_amr.jl"))
+    @test_nowarn test_trixi_include(joinpath(EXAMPLES_DIR, "elixir_euler_khi.jl"))
+  end
+end
```

Comment:
I don't think we ever run the extended tests. Has anyone of you? I certainly didn't. Thus I think we should think about what we want to do with them...

Reply:
I have never run them, but I was not sure if we wanted them to mimic the old Trixi testing structure. For now I will keep them around but I am okay with removing them in the future if they are deemed unnecessary.

Reply:
OK, sounds reasonable. Tracking this in #243